frontend/src/app/projects/project-detail/model-overview/model-detail/git-model.service.ts

```diff
@@ -94,6 +94,17 @@ export class GitModelService {
       source
     );
   }
+
+  deleteGitSource(
+    project_slug: string,
+    model_slug: string,
+    source: GetGitModel
+  ): Observable<any> {
+    return this.http.delete(
```

Comment:
```suggestion
  ): Observable<void> {
    return this.http.delete<void>(
```
I think `void` is the better type here, since we do not expect a return value.